### DISCUSSION TITLE
Do you have any suggestions for Tools, Plugins or Applications that can manage network, users and groups on AMS?

### DISCUSSION THREAD

**Discussion by williehao** [ant-media/Ant-Media-Server#5844] [FAQs]
1.      Do you have any suggestions for Tools, Plugins or Applications that can manage network, users and groups on AMS? 
 
2.      Does the management function include calculators (like network traffic metrics etc.) or restrictions (like user or group viewing permissions etc.)?

**Comment by tremzo** [comment]
@williehao the web panel enables the management of users and groups (Admin, User, Viewer). Here is the document that explains about user management https://antmedia.io/docs/get-started/user-management/. 

Regarding your other query about calculators, I am not sure I follow what you need. Can you provide some more detail or examples of what you'll like to see?

**Reply by williehao** [reply]
Hi All

Please assist me for following issues
1. How can I restrict some users from viewing the LiveApp or WebRTCAppEE vidoes?
2. How can I restrict some users from viewing the LiveApp or WebRTCAppEE with times or network traffic (ex: under 10 mins or 10G)?
3. Does AMS have a graphical user interface (GUI) to configure user restrictions for the LiveApp or WebRTCAppEE?
4. Does AMS have any APIs that can be used to restrict user access to the LiveApp or WebRTCAppEE?

Ps: In the end we wish we can calculate the network/data traffic from AMS that can integrate into our Web site system.

**Reply by tremzo** [reply]
@williehao If you check the link I provided ( https://antmedia.io/docs/get-started/user-management/), you can see that you can configure users in the web panel and restrict them to only one application e.g. LiveApp so once they login, they will not be able to access WebRTCAppEE application and see any live streams or videos under that application. 

Similarly, you can create a new user with these permissions using the API https://antmedia.io/rest/#/ManagementRestService/addUser.

Please check our documentation on monitoring here https://antmedia.io/docs/category/monitoring-ant-media-instance/ and let us know if this will provide all the necessary data you need. It can monitor network and server health.
